### DISCUSSION TITLE
How to be consistent?

### DISCUSSION THREAD

**Discussion by vishakh-abhayan** [community/community#38093] [Other Feature Feedback, Questions, & Ideas] [answered]
It's hard to be consistent because we tend to focus on the outcome more than the process. Put another way, we're more drawn to the positive feelings of outcomes rather than the struggle of the journey. Most of us quit during the struggle before we can experience the rewards of staying the course.😊

**Comment by david-engelmann** [comment, marked as answer]
People love writing, nobody loves reading

**Reply by vishakh-abhayan** [reply]
only love❤️
